src/io/ipc/gen/Schema.rs

```diff
@@ -782,9 +782,10 @@ impl Type {
     pub const LargeBinary: Self = Self(19);
     pub const LargeUtf8: Self = Self(20);
     pub const LargeList: Self = Self(21);
+    pub const Extension: Self = Self(22);
```

Comment:
we can't change this generated code as other implementations will not match it. extension types are shared in a different way, see comment above.